CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.9.1 - 2018-12-10
+This is a maintenance release.
+
+Maintenance:
+  * (EZ-128) Add `numeric-uid-gid` option to allow for specifying a reserved
+    uid/gid for the 'puppet' user. This option only affects rpm packaging.
```

Comment:
This isn't specifically for the 'puppet' user, just whichever user your ezbake config has the package creating